netbench/scripts/netbench-client.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
```

Comment:
Whatever course we end up taking, we'll definitely want to document how exactly it works on our README.md or somewhere else.